apps/meteor/app/authorization/server/streamer/permissions/index.ts

```diff
@@ -4,7 +4,7 @@ import { Permissions } from '@rocket.chat/models';
 
 Meteor.methods({
 	async 'permissions/get'(updatedAt: Date) {
-		check(updatedAt, Match.Maybe(Date));
+		check(new Date(updatedAt), Match.Maybe(Date));
```

Comment:
updatedAt is supposed to already be a Date

Reply:
Fixed!